### PR TITLE
Jsn/java class

### PR DESCRIPTION
**Pull request by jsarenik**
Fix a couple of `gradlew` build warnings.

### CODE REVIEW

**Comment by lvaccaro** [comment]
thanks to report and fix ack 3c20c12d9b0b496fb3987095c1d51ca7a737ba4f

File: app/src/main/java/com/lvaccaro/lamp/services/TorService.kt [hunk 1]
```diff
@@ -104,7 +104,7 @@ class TorService : IntentService("TorService") {
     }
 
     private fun cancelNotification() {
-        Log.d(this.javaClass.canonicalName, "******** Cancel notification called ********")
+        Log.d(this::class.java.canonicalName, "******** Cancel notification called ********")
```
Comment (vincenzopalazzo) [inline app/src/main/java/com/lvaccaro/lamp/services/TorService.kt:107]: By google line guide `this::class.java.canonicalName` should be inside a global variable TAG, but I think this is my fault.

I will fix it inside this PR

File: app/src/main/java/com/lvaccaro/lamp/services/TorService.kt [hunk 2]
```diff
@@ -125,13 +125,12 @@ class TorService : IntentService("TorService") {
     fun getPid(p: Process): Int {
         var pid = -1
         try {
-            val f: Field = p.javaClass.getDeclaredField("pid")
+            val f: Field = p::class.java.getDeclaredField("pid")
             f.setAccessible(true)
             pid = f.getInt(p)
             f.setAccessible(false)
         } catch (e: Throwable) {
-            pid = -1
```
Comment (vincenzopalazzo) [inline app/src/main/java/com/lvaccaro/lamp/services/TorService.kt:133]: Usually, it isn't good manners left the catch empty in Java and all the other types of language based on java.

Maybe is good to print some log message inside the catch with some good information about the type of error.

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
Ehi @jsarenik, good catch, thank you to update the kotlin language to the next version, this is my fault, in fact, `.javaClass` was deprecated since kotlin 1.1

>As @marstran correctly noted in the comment on the question, .javaClass once was considered to be deprecated (see the Kotlin 1.1 RC announcement) because it can break type safety (see below), but it was afterward left as-is because it was widely used and replacing it with the alternative of ::class.java would require adding explicit unchecked casts in the code.

[Source](https://stackoverflow.com/a/46675407/10854225)

In addition, this PR gives me the opportunity to see that the TorServices doesn't respect the Google guidelines, this is my fault and the request for change is self-assign to me.

Thanks to this PR and to improve the code base :-)

**Pull request merged**